### ISSUE TITLE
Add some compoments

### ISSUE THREAD

**Issue by skanehira** [awesome-gocui/gocui#16]
**Describe the feature you'd like**
I want to add some components into gocui to create modal, form, etc easily

**Describe alternatives you've considered**
I made the components.
How about merging this into gocui?
https://github.com/skanehira/gocui-component

**Comment by mjarkk** [comment]
I think we need to hold the base library minimal, thought the current state of gocui is extremely minimal and some UI helpers / components would be a nice addition.  

I'm oke with merging gocui-component into gocui.

**Comment by glvr182** [comment]
Agreed, keeping the library small is important, but i think adding @skanehira 's components would have more pros than cons

**Comment by Syleron** [comment]
do we not feel that these components are considered essential to any "minimalistic" UI framework?
